libs/sdk-backend-tiger/src/convertors/fromBackend/dateGranularityConversions.ts

```diff
@@ -1,14 +1,17 @@
 // (C) 2019-2021 GoodData Corporation
-import { JsonApiAttributeAttributesGranularityEnum } from "@gooddata/api-client-tiger";
+import {
+    JsonApiAttributeAttributesGranularityEnum,
```

Comment:
The "old" `JsonApiAttributeAttributesGranularityEnum` is still used in one of the functions in this file (GitHub won't let me put a comment there). Should that also be replaced by `RelativeDateFilterBodyGranularityEnum`?

Reply:
Good catch, they are the same enum on the backend, but included in 2 different OAPI schemas. I reverted back one side of the conversion and now it makes more sense. This will work for now, ee will need to however change that in the future, I verified that with @To-da .

Reply:
Ok, thanks :)